app/models/cart.rb

```diff
@@ -69,35 +69,20 @@ def prepare_all
     reservations
   end
 
-  def reserve_all(user, res_notes = '', request = false) # rubocop:disable all
+  def reserve_all(user, res_notes = '', request = false)
     # reserve all the items in the cart!
     # takes 3 arguments which is the current user, whether or not
     # the equipment should be requested or reserved,
     # and what notes the reservations should be initialized with
     reservations = prepare_all
     message = []
+
     reservations.each do |r|
-      errors = r.validate
       if request
```

Comment:
But it's so fun! :stuck_out_tongue: You're right, though, that makes much more sense, updating now.